prime-router/src/main/kotlin/tokens/AuthenticatedClaims.kt

```diff
@@ -0,0 +1,74 @@
+package gov.cdc.prime.router.tokens
+
```

Comment:
This is the object that holds Okta Claims for a human user.

It has grown big enough to warrant its own file.